pages/Getting Started/Installation.md

```diff
@@ -240,6 +240,24 @@ For detailed instructions on installing this build see
 
 {{% /details %}}
 
+{{% details title="Alpine*" closed="true" %}}
+
+Hyprland is available in Alpine's [testing repository](https://wiki.alpinelinux.org/wiki/Repositories#Testing) which can be enabled in `/etc/apk/repositories` by adding
+
+```plain
+http://dl-cdn.alpinelinux.org/alpine/edge/testing
+```
+
+This will only work on Alpine linux edge, not on any stable release. For use on stable stable releases, see the [Alpine wiki](https://wiki.alpinelinux.org/wiki/Repositories#Using_the_testing_repository_on_stable_branches)
```

Comment:
stable stable?

Reply:
My bad, Fixed

Reply:
Sorry for the messy PR, squashing on mobile us no fun.

Should be all good now